atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/CacheStoreImpl.java

```diff
@@ -78,13 +76,7 @@ public TransactionScopedCache getCache(StartTimestamp timestamp) {
 
     @Override
     public void removeCache(StartTimestamp timestamp) {
-        Optional<Caches> cache = Optional.ofNullable(cacheMap.remove(timestamp));
-
-        if (!cache.isPresent()) {
-            log.warn(
-                    "Attempted to remove cache state, but no cache was present for timestamp",
-                    SafeArg.of("timestamp", timestamp));
-        }
+        cacheMap.remove(timestamp);
```

Comment:
Just to be 100% sure that this fixes the initial issue, can we keep this log line? Actually, if would great if we add another log line when we successfully remove from cache, keeping ts in params, so we know if the logs we are seeing on the dev stack are due to double removal attempts

Reply:
We could do. I don't have strong feelings about keeping/removing this log line (at least for now), but adding the second logline should maybe go on an RC

Reply:
re-added the old logs

Reply:
Could log removal at debug

Reply:
I don't think this log itself is very useful